### PR TITLE
Move your_first_game to a dedicated section, write index.rst

### LINKED ISSUE #4686: Split your first 2d game into multiple lessons
- [x] Create a section dedicated to dodge the creeps 2D
- [x] Split the long tutorial into multiple pages

### PR DESCRIPTION
**Pull request by NathanLovato**
Working on splitting the Your first 2D game series.

Closes #4686

### CODE REVIEW

**Comment by NathanLovato** [comment]
I split the page into a series, added 4 pictures to the first part and moved a bit of text to the intro page.

**Comment by cbscribe** [comment]
The separation makes sense.

The sections end fairly abruptly. I think we need a breadcrumb at the end of each section leading directly to the next, such as "Click here to move on to the next step." It might also be helpful to briefly (one or two sentences) summarize what we've just done; sort of a checkpoint before moving on. I find a lot of questions/problems involve users who've moved past a step without checking that they'd completed it, then aren't sure where they went wrong.

**Comment by NathanLovato** [comment]
I'll add some text to better link the sections.

Regarding the breadcrumbs, do you mean adding a link that leads to the same page as the next button?

**Comment by NathanLovato** [comment]
Just added the transition text at the end of lessons. Let me know if you'd still like me to add links on top of the buttons. I haven't noticed people had trouble with that personally, more like as you said, just skipping or more often missing steps.

File: getting_started/first_2d_game/03.coding_the_player.rst [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+
+ .. code-tab:: csharp
+
+    using Godot;
+    using System;
+    
+    public class Player : Area2D
+    {
+        [Export]
+        public int speed = 400; // How fast the player will move (pixels/sec).
```
Comment (paulloz) [inline getting_started/first_2d_game/03.coding_the_player.rst:44]: According to the [style guide](https://docs.godotengine.org/en/latest/tutorials/scripting/c_sharp/c_sharp_style_guide.html#naming-conventions), this should be capitalised (same for `screenSize` below).

File: getting_started/first_2d_game/05.the_main_game_scene.rst [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+ .. code-tab:: csharp
+
+    public class Main : Node
+    {
+        // Don't forget to rebuild the project so the editor knows about the new export variable.
+
+    #pragma warning disable 649
+        // We assign this in the editor, so we don't need the warning about not being assigned.
+        [Export]
+        public PackedScene mobScene;
```
Comment (paulloz) [inline getting_started/first_2d_game/05.the_main_game_scene.rst:96]: According to the [style guide](https://docs.godotengine.org/en/latest/tutorials/scripting/c_sharp/c_sharp_style_guide.html#naming-conventions), this should be capitalised (same for `score` below).

File: getting_started/first_2d_game/03.coding_the_player.rst (hunk 1 above)
Comment (NathanLovato) [reply getting_started/first_2d_game/03.coding_the_player.rst:44]: Please feel free to edit the csharp code in a separate pull request. I won't touch the existing csharp code, I'm just splitting the existing one-page tutorial

Comment (paulloz) [reply getting_started/first_2d_game/03.coding_the_player.rst:44]: Noted, and will do 👌

Comment (NathanLovato) [reply getting_started/first_2d_game/03.coding_the_player.rst:44]: Thanks, I'll rebase and merge now so you can work on top of these changes

**Comment by NathanLovato** [comment]
I went ahead and merged but stay available to make further changes regarding the split if need be.

File: getting_started/first_2d_game/03.coding_the_player.rst (hunk 1 above)
Comment (pellul) [reply getting_started/first_2d_game/03.coding_the_player.rst:44]: Oops!
I didn't see your conversation and made a PR concerning these changes [here](https://github.com/godotengine/godot-docs/pull/4723).
@paulloz, I hope you haven't started yet

Comment (paulloz) [reply getting_started/first_2d_game/03.coding_the_player.rst:44]: I was packaging that in a bigger PR. But that's no big deal, don't worry.

**Pull request merged**